docs/content/sdk/python/manual.md

```diff
@@ -34,39 +34,47 @@ detailed instructions on locating API Keys, please refer to the
 The created instance can then be used to start a flow:
 
 ```python
-    # business logic produces labels
-    labels = {
-        "key": "value",
-    }
-
-    # start_flow performs a flowcontrol.v1.Check call to Aperture Agent.
-    # It returns a Flow or raises an error if any.
-    flow = aperture_client.start_flow(
-      control_point="AwesomeFeature",
-      explicit_labels=labels,
-      check_timeout=timedelta(seconds=200),
-    )
-
-    # Check if flow check was successful.
-    if not flow.success:
-        logger.info("Flow check failed - will fail-open")
-
-    # See whether flow was accepted by Aperture Agent.
-    if flow.should_run():
-        # do actual work
-    else:
-        # handle flow rejection by Aperture Agent
-        flow.set_status(FlowStatus.Error)
-    flow.end()
+  # business logic produces labels
```

Comment:
@IridiumOxide,
Please use snippets to actual code instead of explicit code in markdown. Refer how snippets work in top level getting started